Changelog.md

```diff
@@ -7,6 +7,7 @@ Language Features:
 
 
 Compiler Features:
+ * Code Generator: Transient storage value type state variables are now supported by the legacy pipeline.
```

Comment:
I guess we should use `default pipeline` ?
```suggestion
 * Code Generator: Transient storage value type state variables are now supported by the default pipeline.
```

Reply:
There's already precedent for us calling it the legacy pipeline, which I do kinda find weird, since it's not yet legacy. @ekpyron @cameel thoughts?

Reply:
> By the way: here we can just in the via-IR PR turn this into "Support for transient storage state variables of value types" period :-).

Changing a previous changelog entry is allowed? (assuming it's for an unreleased version)

Reply:
> There's already precedent for us calling it the legacy pipeline, which I do kinda find weird, since it's not yet legacy.

It kinda is. It's in active use and still the default, but it's in maintenance mode and we do officially consider it legacy. That's what we've been calling it everywhere for ages now, including blog posts, docs and even error messages. If we were to call it something else, we should have proper name for it to distinguish it from the IR pipeline, but it's a bit late to introduce one now so I'd just continue calling it *legacy*.

And I don't think *default* is any better. It's not very descriptive and may quickly become confusing given that we're getting close to flipping the default.

> Changing a previous changelog entry is allowed? (assuming it's for an unreleased version)

In this situation, yeah. If a single feature (from user's perspective) is spread over several PRs, subsequent PRs can just amend the same changelog entry. No need to artificially have one entry per PR when a single one makes more sense.